README.md

```diff
@@ -16,7 +16,7 @@ This tool supports compiling a workspace for all combinations of the following:
 * Architecture: `armhf`, `aarch64`, `x86_64`
 * ROS Distro
   * ROS: `melodic`, `noetic`
-  * ROS 2: `dashing`, `foxy`, `galactic`, `rolling`
+  * ROS 2: `foxy`, `galactic`, `humble`, `rolling`
```

Comment:
Note that support for Galactic will officially end tomorrow (end of November 2022), so we could just remove it. Same thing in the other file.